### ISSUE TITLE
Inconsistant Metrics

### ISSUE THREAD

**Issue by tuvovan** [sczhou/LEDNet#21]
Hello,

I'm trying to get the metrics shown in the paper (NRQM NIQE) but could not. 
I used pyiqa library and tested with 3 different weight files provided in this repo.

Would you happen to know why? Thank you!

**Comment by Yolo1-gguo** [comment]
> Hello,
> 
> I'm trying to get the metrics shown in the paper (NRQM NIQE) but could not. I used pyiqa library and tested with 3 different weight files provided in this repo.
> 
> Would you happen to know why? Thank you!

May I ask if you have solved it？I have a question to ask you: How do you test the indicators such as MUSIQ, NRQM and NIQE? Is it dependent on the pyiqa library

**Comment by tuvovan** [comment]
I downloaded the official code online and follow their instructions.

**Comment by Yolo1-gguo** [comment]
![Image](https://github.com/user-attachments/assets/783c86bf-1ad9-4f00-a504-3d408f9772b4)Hi, how did you modify the contents of the metrics file during testing?

**Comment by tuvovan** [comment]
The tip was, generate all your images, save them into 1 folder, and then download the code of MUSIQ, NRQM and NIQE or any metric you want and then follow their README to run.

**Comment by Yolo1-gguo** [comment]
> 提示是，生成所有图像，将它们保存到 1 个文件夹中，然后下载 MUSIQ、NRQM 和 NIQE 的代码或您想要的任何指标，然后按照他们的 README 运行。

Thank you very much for your reply!

**Comment by Yolo1-gguo** [comment]
Can you tell me if the pre-training patterns you used for testing on Real-LOL-Blur were trained on LOL-Blur?
